packages/functional-tests/src/app.ts

```diff
@@ -76,16 +80,22 @@ export default class App {
     private userLeft = (user: MRESDK.User) => {
         console.log(`user-left: ${user.name}, ${user.id}`);
     }
-    private async startTest(testName: string,  user: MRESDK.User) {
+    private async startTest(testName: string, user: MRESDK.User) {
         if (this.activeTests[testName]) {
             console.log(`Test already running: '${testName}'`);
         } else if (!this.testFactories[testName]) {
             console.log(`error: Unrecognized test: '${testName}'`);
         } else {
-            const test = this.activeTests[testName] = this.testFactories[testName](user);
+            const test = this.activeTests[testName] = this.testFactories[testName]();

```

Comment:
Did you lose the `user` parameter here?

Reply:
Oh I think this is correct.

Reply:
that's because it's part of the commit being reverted. it was stuff that came as part of your commit that is being reverted